rosbag2_compression/src/rosbag2_compression/sequential_compression_reader.cpp

```diff
@@ -67,7 +67,7 @@ void SequentialCompressionReader::preprocess_current_file()
      * Because we have no way to check whether the bag was written correctly,
      * check for the existence of the prefixed file as a fallback.
      */
-    const rcpputils::fs::path base{base_folder_};
+    rcpputils::fs::path base{base_folder_};
```

Comment:
This _should_ be allowed to be const, but that was fixed in https://github.com/ros2/rcpputils/pull/119 which is API-changing and therefore has not been backported to Foxy